### PR TITLE
Fix string fragment issues

### PR DESCRIPTION
**Pull request by corwintines**
Fix string fragments outlined in notion card:

- [https://github.com/ethereum/ethereum-org-website/blob/b4121564662828caa940302bb5591e2968faa0bf/src/intl/en/page-wallets-find-wallet.json#L132](https://github.com/ethereum/ethereum-org-website/blob/b4121564662828caa940302bb5591e2968faa0bf/src/intl/en/page-wallets-find-wallet.json#L132)
- [https://github.com/ethereum/ethereum-org-website/blob/4e8f8826890e949212db94d94fdec1186f36c6c0/src/intl/en/page-eth2-vision.json#L45](https://github.com/ethereum/ethereum-org-website/blob/4e8f8826890e949212db94d94fdec1186f36c6c0/src/intl/en/page-eth2-vision.json#L45)
- [https://github.com/ethereum/ethereum-org-website/blob/c1e1a26f118b8541d674320ac038761a0003bfb8/src/intl/en/page-eth2-get-involved-bug-bounty.json#L42](https://github.com/ethereum/ethereum-org-website/blob/c1e1a26f118b8541d674320ac038761a0003bfb8/src/intl/en/page-eth2-get-involved-bug-bounty.json#L42)

### CODE REVIEW

File: src/components/Translation.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from "react"
-import { FormattedMessage } from "gatsby-plugin-intl"
+import { FormattedHTMLMessage } from "gatsby-plugin-intl"
```
Comment (minimalsm) [inline src/components/Translation.js:2]: Nice!

File: src/intl/bg/page-developers-tutorials.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   "page-tutorial-submit-btn": "Публикувайте урок",
   "page-tutorial-submit-tutorial": "За да публикувате урок, ще трябва да използвате GitHub. Добре дошли сте да зададете въпрос или заявка за сливане.",
   "page-tutorial-subtitle": "Добре дошли в нашия подбран списък с уроци за общността.",
-  "page-tutorial-tags-error": "Никои от уроците нямат всички тези тагове",
+  "page-tutorial-tags-error": "Никои от уроците нямат всички тези тагове <b>все още</b>",
```
Comment (minimalsm) [inline src/intl/bg/page-developers-tutorials.json:18]: No biggie but for a11y I think `<strong>` is more semantic than `<b>` in this usage

File: src/intl/en/page-developers-tutorials.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
   "page-tutorial-submit-tutorial": "To submit a tutorial, you'll need to use GitHub. We welcome you to create an issue or a pull request.",
   "page-tutorial-subtitle": "Welcome to our curated list of community tutorials.",
   "page-tutorial-tags-error": "There are no tutorials with all your chosen tags yet",
-  "page-tutorial-title": "Ethereum Development Tutorials",
+  "page-tutorial-title": "Ethereum Development Tutorials <b>yet</b>",
```
Comment (minimalsm) [inline src/intl/en/page-developers-tutorials.json:19]:
```suggestion
  "page-tutorial-title": "Ethereum Development Tutorials",
```

**Review by minimalsm — COMMENTED** [review]
Nice find on the FormattedHTMLMessage 😀

Not exactly tied to this PR but re-iterating my concern that adding a translated equivalent of the word 'yet' to languages with a potentially different sentence structure might lead to some confusion (will default to @lukassim on this).

File: src/intl/en/page-developers-tutorials.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
   "page-tutorial-submit-tutorial": "To submit a tutorial, you'll need to use GitHub. We welcome you to create an issue or a pull request.",
   "page-tutorial-subtitle": "Welcome to our curated list of community tutorials.",
   "page-tutorial-tags-error": "There are no tutorials with all your chosen tags yet",
```
Comment (minimalsm) [inline src/intl/en/page-developers-tutorials.json:18]:
```suggestion
  "page-tutorial-tags-error": "There are no tutorials with all your chosen tags <b>yet</b>",
```

**Comment by corwintines** [comment]
@minimalsm changed all the <b> to <strong>. Thanks for review.

**Comment by minimalsm** [comment]
**@corwintines very welcome**

**Comment by corwintines** [comment]
Dangit, didnt realize github would jsut consume that html haha

**Review by minimalsm — APPROVED** [review]

**Review by wackerow — APPROVED** [review]
This is game changing.

**Comment by lukassim** [comment]
> Not exactly tied to this PR but re-iterating my concern that adding a translated equivalent of the word 'yet' to languages with a potentially different sentence structure might lead to some confusion (will default to @lukassim on this).

This should be fine with this approach, as long as 'yet' is part of the same string as the rest of the sentence. This way, the translators can move the tags to a suitable part of the sentence.

**Pull request merged**